packages/mui-material/src/Accordion/Accordion.test.js

```diff
@@ -169,6 +169,7 @@ describe('<Accordion />', () => {
             Accordion.propTypes,
             {
               classes: {},
+              // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
I wonder if we should disable this rule globally. We use fragments in a couple of places on purpose (and because of the limitations of other tools).

Reply:
OK, let's leave it for now. If we end up having to disable it more, we'll do it globally.